### DISCUSSION TITLE
[GENERAL] Transmog display - azerothcore

### DISCUSSION THREAD

**Discussion by V3nomAnc3r** [azerothcore/azerothcore-wotlk#3955] [General]
Hey can some one update transmog display for azerothcore

https://github.com/Rochet2/TrinityCore/tree/transmogvendor_3.3.5/src/server/scripts/Custom/TransmogDisplayVendor

**Comment by Helias** [comment]
Maybe the author? @Rochet2? 

Probably is better to move this thread/issue on the mod-transmog repo.

**Comment by Rochet2** [comment]
I probably will not.
And its not just update, but conversion.

**Comment by BarbzYHOOL** [comment]
i don't even understand what he means

but yes, our transmog is a bit older than the TC one

**Comment by Winfidonarleyan** [comment]
Add 3-5 hooks, create module and done

**Comment by BarbzYHOOL** [comment]
create module?

**Comment by Winfidonarleyan** [comment]
after added new hook for this

**Comment by BarbzYHOOL** [comment]
add to existing module
